javaslang/src-gen/main/java/javaslang/CheckedFunction1.java

```diff
@@ -79,7 +79,7 @@
      * @return a function that applies arguments to the given {@code partialFunction} and returns {@code Some(result)}
      *         if the function is defined for the given arguments, and {@code None} otherwise.
      */
-    static <T1, R> Function1<T1, Option<R>> lift(CheckedFunction1<T1, R> partialFunction) {
+    static <T1, R> Function1<T1, Option<R>> lift(CheckedFunction1<? super T1, R> partialFunction) {
```

Comment:
I think we will also need to widen the result type R: `? extends R` for al occurrences (lift, liftTry, recover).

Could you also widen the parameters (`? super`for args and `? extends` for return type) for the method `of(...)` of CheckedFunction and Function?